webview/src/plots/components/GetStarted.tsx

```diff
@@ -5,13 +5,19 @@ import { StartButton } from '../../shared/components/button/StartButton'
 
 export type AddPlotsProps = {
   hasUnselectedPlots: boolean
+  hasNoCustomPlots: boolean
 }
 
 export const AddPlots: React.FC<AddPlotsProps> = ({
-  hasUnselectedPlots
+  hasUnselectedPlots,
+  hasNoCustomPlots
 }: AddPlotsProps) => (
   <div>
-    <p>No Plots to Display.</p>
+    <p>
+      {hasNoCustomPlots
+        ? 'No Plots to Display'
+        : 'No Selected Plots or Experiments'}
```

Comment:
[I] I think maybe don't switch the text, use "No Plots to Display" for everything. Here is an example where there is an experiment selected and no plots:

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/37993418/229389778-e3e55170-92a4-44a9-8ee1-a0027063fe06.png">

Reply:
Updated the text!